.circleci/config.yml

```diff
@@ -2076,6 +2092,7 @@ workflows:
       - smoke-standalone-26
       - smoke-standalone-27
       - smoke-standalone-35
+      - smoke-standalone-35-rate-limit
```

Comment:
IIRC, each new smoke tests job adds quite some overhead due to image caching, etc.

It may make sense to combine it with some other existing standalone target. (e.g. just run two tox targets as part of the existing ``smoke-standalone-35`` job).

@ArthurKamalov what do you think?

Reply:
> IIRC, each new smoke tests job adds quite some overhead due to image caching, etc.
I think standalone smoke tests do not use image caching, but I also think it is possible to combine them.

PS: Sorry, I had a tons of notifications from other repos and I missed this.